<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectionModelGUI</class>
 <widget class="QWidget" name="SectionModelGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="newPButton">
       <property name="text">
        <string>Nuovo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delPButton">
       <property name="text">
        <string>Canc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
